### ISSUE TITLE
semicolon comment-start

### ISSUE THREAD

**Issue by TLINDEN** [TLINDEN/config-general-mode#3]
; seems to be considered as comment-start.

**Comment by TLINDEN** [comment]
```
             position: 1676 of 3950 (42%), column: 68
            character: ; (displayed as ;) (codepoint 59, #o73, #x3b)
    preferred charset: ascii (ASCII (ISO646 IRV))
code point in charset: 0x3B
               script: latin
               syntax: < 	which means: comment
             category: .:Base, >:Not at bol, a:ASCII, l:Latin, r:Roman
             to input: type "C-x 8 RET 3b" or "C-x 8 RET SEMICOLON"
          buffer code: #x3B
            file code: #x3B (encoded by coding system undecided-unix)
              display: by this font (glyph code)
    xft:-PfEd-DejaVu Sans Mono-normal-normal-normal-*-12-*-*-*-m-0-iso10646-1 (#x1E)

Character code properties: customize what to show
  name: SEMICOLON
  general-category: Po (Punctuation, Other)
  decomposition: (59) (';')

There are text properties here:
  face                 font-lock-comment-face
  fontified            t

```

... for whatever reason

**Comment by TLINDEN** [comment]
fixed with commit b4a8e6b  .
